pygmt/tests/test_grdimage.py

```diff
@@ -93,9 +93,32 @@ def test_grdimage_over_dateline(xrgrid):
 
 
 @check_figures_equal()
-def test_grdimage_central_longitude(grid, fig_ref, fig_test):
+@pytest.mark.parametrize("lon0", [0, 123, 180])
+@pytest.mark.parametrize("proj_type", ["H", "W"])
+def test_grdimage_central_meridians(grid, proj_type, lon0, fig_ref, fig_test):
     """
-    Test that plotting a grid centred at different longitudes/meridians work.
+    Test that plotting a grid with different central meridians (lon0) using
+    Hammer (H) and Mollweide (W) projection systems work.
     """
-    fig_ref.grdimage("@earth_relief_01d_g", projection="W120/15c", cmap="geo")
-    fig_test.grdimage(grid, projection="W120/15c", cmap="geo")
+    fig_ref.grdimage(
+        "@earth_relief_01d_g", projection=f"{proj_type}{lon0}/15c", cmap="geo"
+    )
+    fig_test.grdimage(grid, projection=f"{proj_type}{lon0}/15c", cmap="geo")
```

Comment:
This test checks that #515 is fixed, specifically, that PyGMT can plot xarray grids using the Hammer (H) (and Mollweide (W)) projection system on central longitudes 0, 123 and 180deg.